mods/cnc/rules/defaults.yaml

```diff
@@ -153,6 +153,7 @@
 		GenericName: Helicopter
 	WithFacingSpriteBody:
 	WithShadow:
+		Offset: 43, 43, -85
```

Comment:
Putting the shadows below ground level isn't great and may cause rendering glitches as we progress further with zbuffer support.  Can you move that offset into Y instead?